docs/manual/src/docs/asciidoc/_includes/servlet/authorization/method-security.adoc

```diff
@@ -6,6 +6,213 @@ It provides support for JSR-250 annotation security as well as the framework's o
 From 3.0 you can also make use of new <<el-access,expression-based annotations>>.
 You can apply security to a single bean, using the `intercept-methods` element to decorate the bean declaration, or you can secure multiple beans across the entire service layer using the AspectJ style pointcuts.
 
+=== EnableMethodSecurity
+
+In 5.6, we can enable annotation-based security using the `@EnableMethodSecurity` annotation on any `@Configuration` instance.
+
+[NOTE]
```

Comment:
Similar to how how we use `{` for if blocks, we use `====` for [admonitions](https://docs.asciidoctor.org/asciidoc/latest/blocks/admonitions/)